presence/fpp/fpp/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+mod fspl_converter;
```

Comment:
Some useful lints you can add in your top level `lib.rs` files: 
```
#![deny(
    missing_docs,
    clippy::indexing_slicing,
    clippy::unwrap_used,
    clippy::panic,
    clippy::expect_used
)]
```
This will help make sure your code does not contain any unnecessary panics, and that all public stuff is documented

Reply:
You will also want to run `cargo clippy` which will auto-suggest you code improvements

Reply:
Done.